test/test_group.py

```diff
@@ -83,6 +88,12 @@ def test_edit_group(db, client):
     set_and_assert_group_data(db, client, url_for('edit_group', avg=DEFAULT_GROUP_NAME))
 
 
+@create_package(name='foo', version='1.2.3-4')
+@logged_in
+def test_edit_group_bug_url(db, client):
+    set_and_assert_group_data(db, client, url_for('add_group'), bug_ticket='')
+
```

Comment:
It would be neat if we can have a second test case for [community] package bug ticket creation links